tb/uvmt/uvmt_cv32e40x_tb_files.flist

```diff
@@ -27,6 +27,7 @@ ${DV_UVMT_PATH}/uvmt_cv32e40x_interrupt_assert.sv
 ${DV_UVMT_PATH}/uvmt_cv32e40x_pma_assert.sv
 ${DV_UVMT_PATH}/uvmt_cv32e40x_rvfi_assert.sv
 ${DV_UVMT_PATH}/uvmt_cv32e40x_zc_assert.sv
+${DV_UVMT_PATH}/uvmt_cv32e40x_rv32e_assert.sv
```

Comment:
I guess this avoided trouble before because E asserts were only used in local formal. Nobody ran E in sim.